syntax = "proto3";

import "google/protobuf/any.proto";

package opentera.protobuf;

message TeraMessage {
    google.protobuf.Any message = 1;
}
